lbry/lbry/extras/daemon/storage.py

```diff
@@ -744,3 +752,20 @@ def get_streams_to_re_reflect(self):
             "where r.timestamp is null or r.timestamp < ?",
             int(self.time_getter()) - 86400
         )
+
+    # # # # # # # # # # dht functions # # # # # # # # # # #
+    async def get_peers(self):
+        query = 'select node_id, address, udp_port, tcp_port from peer'
```

Comment:
with the primary key change, this is `select * from peer`

Reply:
Being explicit instead of instead of implicits makes it easier to add a new column, change order and it makes it clear what is being queried.